packages/xo-web/src/xo-app/backup/overview/index.js

```diff
@@ -35,6 +36,18 @@ const jobKeyToLabel = {
   rollingSnapshot: _('rollingSnapshot'),
 }
 
+const _runJob = job =>
+  confirm({
+    title: _('runJob'),
+    body: _('runJobConfirm', {
+      job: (
+        <strong>
+          {job.id.slice(4, 8)} {job.label}
```

Comment:
I think the user would want to see the job tag.
Something like "Are you sure you want to run the \<backup type\> \<id\> (\<tag\>)?" or whatever looks nice and clear enough.